### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by gasparez15** [logicaldoc/community#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by logico01** [comment]
Hola buen dia, soy de bs as argentina y estoy tratando de implementar logicaldoc community en mi oficina particular, el tem de ocr es importante, instale los modulos de la comunidad pero me aparece un error cuando quiero entrar en los ajustes de ocr y ademas no se habilita la opcion de ocr en la parte de documentos, Me podrias ayudar o decirme a donde puedo recurrir?
Gracias y disculpas por las molestias

**Comment by Amirhossein-Nasirizade** [comment]
Hello, I'm having problems launching the project. I'm using Maven 3.9.9 and Java 21. In LogicalDoc's pom.xml (main pom.xml), we have a parent configuration:

<parent>
    <groupId>com.logicaldoc</groupId>
    <artifactId>logicaldoc-parent-pom</artifactId>
    <version>109</version>
</parent>
However, I couldn't locate it and had to commit it, then perform a clean installation of the project. During the clean-install process, I encountered problems with Spring and other dependencies. I appreciate that help me launch the community project?
